assets/js/main.js

```diff
@@ -167,3 +167,34 @@ $('.progress-bar').each(function(index) {
     $(this).toggleClass('dp-open-results')
     $(this).closest('tr').next().toggleClass('show');
   });
+
+// for all functions use dopplerUI namespace
+var dopplerUI = {
+  createDoubleSlide = function(element1, element2){
```

Comment:
At the moment, the current code is ok, but it will give us problems when we decide to add more items to the namespace.

I am not sure about the implementation, but there is a well-known pattern for this. It is something like this:

```suggestion
var dopplerUI  = dopplerUI || {};
dopplerUI.createDoubleSlide = function(element1, element2){
```

Reply:
Or maybe:

```suggestion
var dopplerUI;
(function (dopplerUI) {
    dopplerUI.createDoubleSlide = function(element1, element2){
    // . . .
    };
})(dopplerUI || (dopplerUI = {}));
```

Reply:
@andresmoschini I've done some research regarding this, and it bears no benefit, it keeps the functions within a scope with a more complex syntax. I will leave it with the first suggestion.